src/desktop/components/intercom/index.js

```diff
@@ -1,9 +1,9 @@
 import { data as sd } from "sharify"
 
-window.intercomSettings = {
-  app_id: sd.INTERCOM_APP_ID,
-}
-;(function() {
+export const intercom = function(appID, delayed = false) {
+  window.intercomSettings = {
+    app_id: appID,
+  }
```

Comment:
For the gallery partnerships page, we don't care if the user is logged in or not, so we don't send users to Intercom.

For BNMO, I imagine we will want to talk to both visitors and logged-in users. For logged-in users, we might want to [include additional data](https://www.intercom.com/help/install-on-your-product-or-site/quick-install/install-intercom-on-your-web-app-for-logged-in-users) so we can maintain messages with the same user and manage them on the dashboard. For security reasons, we will then enable [identity verification](https://www.intercom.com/help/configure-intercom-for-your-product-or-site/staying-secure/enable-identity-verification-on-your-web-product). [Volt](https://github.com/artsy/volt/blob/fa6acd310ccb5940eb199c072d6d6f63aeb624ce/app/views/layouts/_intercom.haml) is an example where we only talk to logged-in users.

Reply:
I'm not sure what all that additional info would be used for? Maybe we can pair on finishing this off, I'm pretty unfamiliar with Intercom, sounds like you know more than I!